collect_app/src/main/java/org/odk/collect/android/formmanagement/metadata/FormMetadataParser.kt

```diff
@@ -30,6 +30,7 @@ object FormMetadataParser {
         val base64RsaPublicKey = submission?.getAttributeValue(null, "base64RsaPublicKey")
         val autoDelete = submission?.getAttributeValue(null, "auto-delete")
         val autoSend = submission?.getAttributeValue(null, "auto-send")
+        val clientEditable = submission?.getAttributeValue("http://www.opendatakit.org/xforms", "client-editable")
```

Comment:
I think the changes to `FormMetadataParser`, `FormMetadata` and their tests can be removed